### PR TITLE
Update dunfell with changes from kirkstone (default branch)

### PR DESCRIPTION
**Pull request by danimtb**
Bring changes from kirkstone branch to dunfell (check image is built with the version taken from the target branch of the PR)

### CODE REVIEW

File: recipes-devtools/python3-packaging_20.8.bb [hunk 1]
```diff
@@ -1,13 +0,0 @@
-DESCRIPTION = "Core utilities for Python packages"
```
Comment (danimtb) [inline recipes-devtools/python3-packaging_20.8.bb:1]: we may need this file when we add the conan layer to the image build for dunfell, just to keep that in mind

File: recipes-devtools/python3-deprecation_2.1.0.bb [hunk 2]
```diff
@@ -7,11 +7,11 @@ LIC_FILES_CHKSUM = "file://${S}/LICENSE;md5=e3fc50a88d0a364313df4b21ef20c29e"
 SRC_URI[md5sum] = "6b79c6572fb241e3cecbbd7d539bb66b"
 SRC_URI[sha256sum] = "72b3bde64e5d778694b0cf68178aed03d15e15477116add3fb773e581f9518ff"
 
-RDEPENDS_${PN} = "\
```
Comment (danimtb) [inline recipes-devtools/python3-deprecation_2.1.0.bb:10]: there is some syntax changing here as well

Comment (uilianries) [reply recipes-devtools/python3-deprecation_2.1.0.bb:10]: Yes, yocto started to use a different way.

**Comment by uilianries** [comment]
@danimtb we have a limitation with the CLA. Should we squash force to keep only a single author?

**Comment by danimtb** [comment]
> @danimtb we have a limitation with the CLA. Should we squash force to keep only a single author?

yes, no problem

**Pull request merged**